src/dtgtk/range.c

```diff
@@ -483,6 +483,10 @@ static void _popup_date_update(GtkDarktableRangeSelect *range, GtkWidget *w)
 
   pop->internal_change++;
 
+  dt_bauhaus_combobox_clear(pop->type);
+  dt_bauhaus_combobox_add(pop->type, _("fixed"));
+  if(w == range->entry_min || w == range->entry_max) dt_bauhaus_combobox_add(pop->type, _("relative"));
```

Comment:
In this case I think we can improve a bit more. If a single entry on the combo it seems strange to have it active and when clicking on it see only a single value. I would therefore make the widget non active when a single value is in the combo.

Reply:
done